### PR TITLE
Parser help searching

### PR DESCRIPTION
**Pull request by nids2001**
Make parser help dialog searchable

### CODE REVIEW

**Comment by dcwhite** [comment]
@nids2001 Build error: 
```/Users/runner/work/SCIRun/SCIRun/src/Interface/Modules/Base/HasParserHelpDialog.cc:73:3: error: use of undeclared identifier 'prevResult'
  prevResult = false;
  ^
1 error generated.```

**Comment by dcwhite** [comment]
Will close #1534

File: src/Interface/Modules/Base/HasParserHelpDialog.cc [hunk 1]
```diff
@@ -33,8 +33,7 @@ using namespace SCIRun::Gui;
 
 void ModuleDialogWithParserHelp::popUpParserHelp()
 {
-  if (!help_)
-    help_ = new ParserHelpDialog(this);
+  help_ = new ParserHelpDialog(this);
```
Comment (dcwhite) [inline src/Interface/Modules/Base/HasParserHelpDialog.cc:36]: This line (creating a new instance every time) leads to multiple dialogs being created each time the user clicks the show button. If you add back the null check, that will take care of only creating and showing one instance per module.

**Review by dcwhite — CHANGES_REQUESTED** [review]

**Review by dcwhite — APPROVED** [review]

**Comment by dcwhite** [comment]
👍

**Review by jessdtate — APPROVED** [review]

**Pull request merged**